engine/schema/src/main/java/com/cloud/storage/dao/GuestOSDaoImpl.java

```diff
@@ -62,4 +71,41 @@ public GuestOSVO findByCategoryIdAndDisplayNameOrderByCreatedDesc(long categoryI
         }
         return null;
     }
+
+    /**
+     "select * from guest_os go2 where display_name in"
+     +       "(select display_name from"
+     +               "(select display_name, count(1) as count from guest_os go1 group by display_name having count > 1) tab0)";
+     *
+     * @return
+     */
+    @Override
+    @DB
+    public Set<String> findDoubleNames() {
+        String selectSql = "(select display_name from (select display_name, count(1) as count from guest_os go1 group by display_name having count > 1) tab0)";
```

Comment:
```suggestion
        String selectSql = "select display_name, count(1) as count from guest_os where removed IS NULL group by display_name having count > 1";
```

Reply:
not quite, I want a list of guestos names returned and no counts. I will add the `removed IS NULL`

Reply:
> not quite, I want a list of guestos names returned and no counts. I will add the `removed IS NULL`

@DaanHoogland 
I know. but you have the following code, so it does not matter if count is returned.
```
names.add(rs.getString(1));
```

Reply:
I let the DB do what it is good at. makes sense?

Reply:
it is not needed to use 2 select, imho.
If you do not want to change it, it is ok.